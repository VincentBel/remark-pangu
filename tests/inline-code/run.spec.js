runSpec(__dirname)
runSpec(__dirname, {
    inlineCode: false
})