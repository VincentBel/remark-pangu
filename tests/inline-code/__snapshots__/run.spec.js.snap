// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inline-code.md: inline-code.md 1`] = `
\`abc123\`

\`我a我1\`
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
\`abc123\`

\`我 a 我 1\`

`;
